meetings/2018-01-10.md

```diff
@@ -0,0 +1,117 @@
+* Gibson Fahnestock @gibfahn (observer)
+* James M Snell @jasnell (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Michaël Zasso @targos (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+### Announcements
+* 8.9.4 out
+* Planning for 10.X is out. Try to plan major changes before end of Feb as we'll cut
+  10.x branch then.
+* Release group is discussing creating a more concrete schedule for releases  
+
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/TSC
+* Mentorship initiative ramping up. [#443](https://github.com/nodejs/TSC/issues/443)
+  * Just wanted to make sure everybody is aware
+* Enterprise Advisory Group [#431](https://github.com/nodejs/TSC/issues/431)
+  * James: Heads up at this point. Goal is to look at things happening core and provide
+    feedback. TSC members to be invited as observers as opposed to members.
+  * James: Goal is to bring in companies who have made significant contributions to Node.js
+    and get them a channel for giving their input.
+  * Franziska: who is responsible for receiving and acting on the feedback.
+  * James: will attend as observer and will provide feedback, other TSC members are 
+    free to participate as observer as well.
+   * Franziska: should we clarify in the charter that there is no commitment on the TSC's part.
+   * Rich: concern that there is a real focus on feedback to TSC, may be sending wrong signals
+     about decision making (most decisions are not made by the TSC), and many of the TSC
+     members were surprised as the minutes were written in a way that suggested that the TSC
+     was being spoken for when most TSC members weren't aware of this initiative 
+   * Fishrock: TLDR, important to make sure it is clarified in doc.
+   * James: purely a channel for that group to indicate what things are a priority
+   * Franziska: since its a CommComm initiative, how about less focus in the charter on the TSC
+   * Agreed to take back to GitHub and discuss there.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * N-API
+    * Making progress on satisfying criteria to exit experimental status.
+      * Bcrypt ported and being used in production soon
+      * weak-napi published as alternative to node-weak
+      * Module used in dev-edge production site ported and working to have it be used in
+        production there
+      * Working with maintainer of node-pre-gyp to define/document how to use N-API modules
+        with it.
+    * Moderation
+      * One user permanently banned for wiki vandalism.
+      * Myles dealt with a spam-bot invasion on #node-dev IRC channel
+      * One user.s comment was moderated by a Collaborator, moderation group called into 
```

Comment:
@mhdawson Any idea why this sort of typo (_user.s_ instead of _user's_) seems to happen a lot in your docs? I imagine it's a keyboard or charset thing? Anything that can be done about it?